src/utilities/svgs.js

```diff
@@ -37,6 +43,14 @@ export const svgs = {
     maintenance: TabMaintenanceIconPath,
     rules: TabRulesIconPath
   },
+  devices: {
+    default: DeviceDefaultIconPath,
```

Comment:
Rename `default` icon to `generic` It may be that in some use cases other icons are used as defaults so `generic` is more descriptive in telling you what the icon is rather than how it is used.